### PR TITLE
Bump version to v1.15.0-rc1

### PR DESCRIPTION
**Pull request by agarciamontoro**
#### Summary
Preparing release v1.15.0-RC1

#### Ticket Link
--

### CODE REVIEW

File: config/deployer.sample.json [hunk 1]
```diff
@@ -60,7 +60,7 @@
   "AdminEmail": "sysadmin@sample.mattermost.com",
   "AdminUsername": "sysadmin",
   "AdminPassword": "Sys@dmin-sample1",
-  "LoadTestDownloadURL": "https://github.com/mattermost/mattermost-load-test-ng/releases/download/v1.9.1/mattermost-load-test-ng-v1.9.1-linux-amd64.tar.gz",
```
Comment (streamer45) [inline config/deployer.sample.json:63]: Is this out of date somehow? Noticing the bump from v1.9 to v1.15.

Comment (agarciamontoro) [reply config/deployer.sample.json:63]: Yeah, this is something we need to fix process-wise. We only bump the version in the release branches, but never on master. We may want to merge into master and then cut the branch? :thinking:

Comment (agarciamontoro) [reply config/deployer.sample.json:63]: > Yeah, that would be my expectation to be honest.

Let's test this now

Comment (streamer45) [reply config/deployer.sample.json:63]: That could also work, at which point the pinning would make sense in the release branch only.

**Comment by agarciamontoro** [comment]
Let's modify the process so that we merge this into master and *then* we cut the branch. Merging into master now.

**Review by streamer45 — APPROVED** [review]

**Pull request merged**